mercantile/__init__.py

```diff
@@ -619,25 +642,30 @@ def bounding_tile(*bbox, **kwds):
     """
     if len(bbox) == 2:
         bbox += bbox
+
     w, s, e, n = bbox
+
     truncate = bool(kwds.get("truncate"))
+
     if truncate:
         w, s = truncate_lnglat(w, s)
         e, n = truncate_lnglat(e, n)
-    # Algorithm ported directly from https://github.com/mapbox/tilebelt.
 
     try:
-        tmin = tile(w, s, 32, truncate=truncate)
-        tmax = tile(e, n, 32, truncate=truncate)
+        tmin = tile(w, n, 32, truncate=truncate)
```

Comment:
@sgillies this is the only logic change I see here that doesn't seem to follow from the above tweaks to use epsilon values and consolidate global variables.  Was this a logic error in the previous version you resolved here?

Reply:
The tiling origin is at the upper left, so it is more strictly correct to use (w, n) and (e, s) than what we had before. The algorithm for finding the bounding zoom level isn't sensitive to the order.